.github/dependabot.yml

```diff
@@ -1,20 +1,11 @@
 version: 2
 updates:
 
-  # Maintain dependencies for npm
```

Comment:
We can keep the block and just add `open-pull-requests-limit: 0`
That will lead to security updated only and disable regular version updates.

See: https://docs.github.com/en/enterprise-server@3.3/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates